### PR TITLE
Add fun fact

### PR DESCRIPTION
**Pull request by SaraDawner2000**
Added a place I recommend checking out :) it's free and open access

### CODE REVIEW

**Review by aizatibraimova — APPROVED** [review]
Looks amazing

File: README.md [hunk 1]
```diff
@@ -102,3 +102,15 @@ Clone this repository (or open in codespace), create a branch with your name, ad
 
 | Name                 | Fun Fact |
 | -------------------- | --- |
+
```
Comment (aizatibraimova) [inline README.md:105]: too many lines

**Review by aizatibraimova — APPROVED** [review]

**Review by jrzricardo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (SaraDawner2000) [reply README.md:105]: Thank you for the feedback!

**Review by jrzricardo — APPROVED** [review]
amazing

**Review by aizatibraimova — APPROVED** [review]
Looks good

**Review by jrzricardo — APPROVED** [review]
noice

**Review by APatel-AI — APPROVED** [review]
looks good

**Pull request merged**